### ISSUE TITLE
No blank node content in downloaded concept

### ISSUE THREAD

**Issue by kouralex** [NatLibFi/Skosmos#732]
## At which URL did you encounter the problem?
http://skosmos.dev.finto.fi/euro/en/page/12
## What steps will reproduce the problem?
1. Go to link above
2. Download concept as Turtle

## What is the expected output? What do you see instead?
I'd expect to have blank node content in the file.

**Comment by kouralex** [comment]
Related to issues #420 and #730.

**Comment by kouralex** [comment]
I'd suggest a [CBD-style](https://www.w3.org/Submission/CBD/) resolution for this issue, maybe without the reification part? What do you think @osma ?

**Comment by osma** [comment]
Agreed. Reification is not necessary. But I'm not sure whether it's even possible to do CBD (recursive traversal of blank nodes) with SPARQL. Maybe if you're clever with path expressions...
Just traversing blank nodes a single step (triple) ahead would probably be enough for most purposes.

**Comment by kouralex** [comment]
It just occurred to me that this may be resolved by implementing a user-definable, per vocabulary based Skosmos predicate where one could define in plain SPARQL the required SPARQL-parts used in querying and constructing the resulting RDF. What do you think @osma ?
